rpcserver.go

```diff
@@ -6817,6 +6863,25 @@ func (r *rpcServer) ForwardingHistory(ctx context.Context,
 			AmtInMsat:   uint64(amtInMsat),
 			AmtOutMsat:  uint64(amtOutMsat),
 		}
+
+		if !req.SkipPeerAliasLookup {
+			resp.ForwardingEvents[i].PeerAliasIn, err =
```

Comment:
nit: just use `aliasIn, err := getRemoteAlias(` to make the formatting nicer. Then do `resp.ForwardingEvents[i].PeerAliasIn = aliasIn` later.